playground/mocks/data/idp/idx/authenticator-expired-password-with-enrollment-authenticator.json

```diff
@@ -17,7 +17,7 @@
 				"form": {
 					"value": [{
 						"name": "passcode",
-						"label": "Enter password",
+						"label": "New password",
```

Comment:
This is the intended change. The mock here was out-of-date. Spoke in detail with @zeusandrade-okta 
 ( and related discussion on the flow with @SamSanjabi-okta ).